crates/zoon/src/style/background.rs

```diff
@@ -1,12 +1,37 @@
 use crate::*;
 
+/// Styling to set the background for an element.
+/// # Example
+/// ```no_run
+/// use zoon::*;
+///
+/// let element = El::new().s(Background::new().color(hsluv!(241.3, 100, 96.6)));
+/// ```
 #[derive(Default)]
 pub struct Background<'a> {
+    /// Css properties used by Zoon to style the element
+    /// with Css.
     static_css_props: StaticCSSProps<'a>,
+    /// Css properties that can be used to customize the background directly.
     dynamic_css_props: DynamicCSSProps,
```

Comment:
Well, I did not have to document the private fields actually but I did. 
I did not know that their visibility could impact how we do document them. I saw right now that we can generate the doc for private items with the flag `--document-private-items -`

The first reason for why I did start documenting these fields was just to show up that under the hood we do use css to actually do the job. Just wanted to make the `styling` less exotic and reassure our users that, if we do use css under the hood then in case the Api is not completed enough they can understand it and extend it easily.

Reply:
I resolve it and keep it this way for now.